helm/coder/values.yaml

```diff
@@ -281,6 +281,9 @@ coder:
     # your cloud and specify it here in production to avoid accidental IP
     # address changes.
     loadBalancerIP: ""
+    # coder.service.className -- The class name of the LoadBalancer. See:
+    # https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class
+    className: ""
```

Comment:
Can this be changed to `loadBalancerClass` to match the field in the k8s yaml?

Reply:
Done